src/simd/config.rs

```diff
@@ -17,18 +17,21 @@ pub trait SIMDInstructionSet {
 // ----------------------------- x86_64 / x86 -----------------------------
 
 pub struct SSE;
+pub struct SSEIgnoreNaN;
 
 impl SIMDInstructionSet for SSE {
     const REGISTER_SIZE: usize = 128;
 }
 
 pub struct AVX2; // for f32 and f64 AVX is enough
```

Comment:
AVX or AVX2? Lets check the accuracy of this.

Reply:
For all datatypes except f32 and f64 we need AVX2, but for single and douple precision floats AVX is sufficient. 
It would indeed be more clean / readable to create a separate AVX struct

Reply:
Clearly updated the docs!